packages/pds/src/db/message-queue/messages.ts

```diff
@@ -0,0 +1,97 @@
+export type AddMember = {
```

Comment:
We should open an issue or something to track how we want to evolve the semantics of these events potentially into lexicon, but I'm +1 on not worrying about it for now

Reply:
👍